src/interpreter/Interpreter.ts

```diff
@@ -78,7 +79,8 @@ export class Interpreter {
       }
 
       model.required = schema.required || model.required;
-
+      
+      interpretItems(schema, model, this);
```

Comment:
Sorry 😅 I didn't check that

Reply:
No worries, my mistake for not adding it 😄